ui/narrative/methods/import_fasta_as_assembly_from_staging/spec.json

```diff
@@ -19,6 +19,17 @@
       "text_options" : {}
     }, 
     {
+      "id" : "min_contig_length",
```

Comment:
you forget to do 'input_mapping'. otherwise, min_contig_length won't pass to params.